### PR TITLE
[IMP] add wizards check payment in MP

### PR DESCRIPTION
**Pull request by maq-adhoc**

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://mergebot.adhoc.ar/plugberry/mercadopago/pull/55).

**Comment by maq-adhoc** [comment]
@roboadhoc r+

**Comment by roboadhoc** [comment]
@maq-adhoc because this PR has multiple commits, I need to know how to merge it:

* `merge` to merge directly, using the PR as merge commit message
* `rebase-merge` to rebase and merge, using the PR as merge commit message
* `rebase-ff` to rebase and fast-forward

**Comment by maq-adhoc** [comment]
@roboadhoc  rebase-merge

**Comment by roboadhoc** [comment]
Merge method set to rebase and merge, using the PR as merge commit message.

**Comment by roboadhoc** [comment]
@maq-adhoc unable to stage: merge conflict

File: payment_mercadopago/wizards/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from . import check_payment
```
Comment (bruno-zanotti) [inline payment_mercadopago/wizards/__init__.py]:
```suggestion
from . import check_payments
```

**Review by bruno-zanotti — CHANGES_REQUESTED** [review]

**Comment by bruno-zanotti** [comment]
@roboadhoc r+

**Comment by fw-bot-adhoc** [comment]
@maq-adhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
- plugberry/mercadopago#59

**Comment by fw-bot-adhoc** [comment]
@maq-adhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
- plugberry/mercadopago#59

**Comment by fw-bot-adhoc** [comment]
@maq-adhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
- plugberry/mercadopago#59

**Comment by fw-bot-adhoc** [comment]
@maq-adhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
- plugberry/mercadopago#59

**Comment by fw-bot-adhoc** [comment]
@maq-adhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
- plugberry/mercadopago#59

**Comment by fw-bot-adhoc** [comment]
@maq-adhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
- plugberry/mercadopago#59

**Comment by fw-bot-adhoc** [comment]
@maq-adhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
- plugberry/mercadopago#59

**Comment by fw-bot-adhoc** [comment]
@maq-adhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
- plugberry/mercadopago#59

**Comment by fw-bot-adhoc** [comment]
@maq-adhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
- plugberry/mercadopago#59

**Comment by fw-bot-adhoc** [comment]
@maq-adhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
- plugberry/mercadopago#59

**Comment by fw-bot-adhoc** [comment]
@maq-adhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
- plugberry/mercadopago#59

**Comment by fw-bot-adhoc** [comment]
@maq-adhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
- plugberry/mercadopago#59

**Pull request closed without merging**